build.sbt

```diff
@@ -51,7 +51,7 @@ lazy val http4s = project.in(file("modules/http4s"))
   .settings(
     name := "fuuid-http4s",
     libraryDependencies ++= Seq(
-      "org.http4s" %% "http4s-dsl" % http4sV % Test
```

Comment:
I need `QueryParamDecoder`, `QueryParameterValue` and `ParseFailure` in `implicits.scala`, does it make sense?

Reply:
I follow now, as previously it didn't even need http4s. All of these are available in `http4s-core`. Could we limit the dependency to core, and only use dsl in the test still.

This would allow this for other dsl interactions like Rho.

Reply:
Yep, it makes sense 👍

Reply:
Done, @ChristopherDavenport